cloud_functions/ads_policy_monitor/google_ads_test.py

```diff
@@ -12,11 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Unit tests for google_ads.py"""
+
 import random
 import unittest
-from unittest.mock import MagicMock, patch
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+from test_data_helper import TEST_AD_GROUP_ASSET_POLICY_DATA
+from test_data_helper import TEST_CAMPAIGN_ASSET_POLICY_DATA
+from test_data_helper import TEST_CUSTOMER_ASSET_POLICY_DATA
+from test_data_helper import TEST_EXPECTED_ASSET_POLICY_REPORT
+from gaarf.report import GaarfReport
+from unittest.mock import MagicMock
```

Comment:
Quite a few imports are here twice.

Reply:
Done, don't know how this happened or how I didn't notice it lol